calico_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021-2022 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build calico_test
```

Comment:
tags are not super easy to use for local testing. Could we move tests to differenet pacakges. Say
`internal/pkg/interdomain`, `internal/pkg/signle`, `internal/pkg/calico`. 

Thoughts?